plugin.yaml

```diff
@@ -204,6 +204,14 @@ node_types:
         description: |
           Enable start (power on) operation for VM.
         default: true
+      postpone_delete_networks:
```

Comment:
This isn't in the feature list. Should it be?

Reply:
It has save functionality level as `wait_ip`